v2/cf-nodejs-logging-support/src/lib/config/config.ts

```diff
@@ -208,4 +209,8 @@ export default class Config {
 
         return index;
     }
+
+    public enableTracing() {
```

Comment:
This only allows to enable tracing, which then implicitly defaults to sap passport.
This will be pretty hard to upgrade later without any breaking changes, so I would like to have the tracing type specified directly in any way feasible.

Reply:
Fixed in [24881f8](https://github.com/SAP/cf-nodejs-logging-support/pull/126/commits/24881f8acc3c7fac34f2f3929c44679df991350a)